tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    /**
+     * Must allow:
+     */
+    "downlevelIteration": true,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "noUnusedParameters": false,
+    "removeComments": false,
+
+    /**
+     * Nice to have:
+     */
+    // "preserveConstEnums": true,
+    // "declaration": true,
+    // "allowUnreachableCode": false,
+    // "pretty": true,
+
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "target": "es5",
```

Comment:
target should be browser TSConfig target, as that's usually the lower version of ECMAScript.